Gems/ROS2/Code/Source/ROS2EditorSystemComponent.h

```diff
@@ -0,0 +1,39 @@
+/*
+* Copyright (c) Contributors to the Open 3D Engine Project.
+* For complete copyright and license terms please see the LICENSE at the root of this distribution.
+*
+* SPDX-License-Identifier: Apache-2.0 OR MIT
+*
+*/
+#pragma once
+
+#include <ROS2SystemComponent.h>
+
+#include <AzToolsFramework/Entity/EditorEntityContextBus.h>
+
+namespace ROS2
+{
+    /// System component for ROS2 editor
+    class ROS2EditorSystemComponent
+        : public ROS2SystemComponent
+        , private AzToolsFramework::EditorEvents::Bus::Handler
```

Comment:
The EditorEvents::Bus isn't actually being handled by this Editor component, so it can be removed.

It looks like the Gem Template adds this EBus handler to the EditorComponent, but doesn't actually implement the Event functions.